DataminrPulse/README.md

```diff
@@ -0,0 +1,292 @@
+      ],
+      "id_str": ""
+    },
+    "link": "https://www.shodan.io/host/1873360165",
+    "media": [],
+    "extraData": {
+      "displaySourceName": "Shodan"
+    }
+  },
+  "referenceTerms": [
+    {
+      "term": "remote access systems"
+    },
+    {
+      "term": "port"
+    },
+    {
+      "term": "ip"
+    },
+    {
+      "term": "vnc"
+    },
+    {
+      "term": "sensor"
+    },
+    {
+      "term": "biglobe"
+    }
+  ],
+  "watchlistsMatched": [
+    "3999781"
+  ],
+  "alertType": {
+    "id": "alert",
+    "name": "Alert"
+  },
+  "companies": [
+    {
+      "name": "BIGLOBE Inc.",
+      "topicType": "company",
+      "id": "ywmgww5ibebrbnxid7kc6kve4a",
+      "idStr": "ywmgww5ibebrbnxid7kc6kve4a",
+      "retired": false
+    }
+  ],
+  "image_url": null,
+  "source_type": [
+    "sensor"
+  ],
+  "source_link": "https://www.shodan.io/host/1873360165"
+}
+
+```
+
+### Query Sample
+```sql title = "Alerts by Severity"
+_sourceCategory="new_security" 
+| json "index","odsStatus.source.channels","location.latitude","location.longitude","topics[*].name","categories[*].name","alertType.name","headline","timestamp","watchlistsMatchedByType[*].name","watchlistsMatchedByType[*].userProperties.uiListType","headlineData.via","publisherCategory.name" as alert_id,channel,latitude,longitude,company_list,category_list,severity,caption,event_time,watchlists_name_list,watchlist_cyber_alerts,headline_source,publisher_category nodrop
+// topics array of name (topics[*].name) is the list of companies of alerts.
+// categories array of name (categories[*].name) is the list of topics of alerts.
+| extract field=watchlists_name_list "\"?(?<watchlist_name>[\w\s\,&*.</>!]*)\"?[,\n\]]" multi
+| extract field=channel "\"?(?<channel>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| extract field=caption "via (?<source>.*)\.$" multi
+| where !isNull(channel)
+| where !isNull(source)
+
+// global filters
+| 0 as priority 
+| count by alert_id,severity,priority
+| count as frequency by severity,priority
+| if(severity="Flash",4,if(severity="Urgent",3,if(severity="Urgent Update",2,if(severity="Alert",1,0)))) as priority
+
+| sort by priority
+| fields - priority
+```
+
+## Set up Collection
+
+Dataminr Pulse Alerts will be forwarded to the Sumo Logic using the Dataminr Pulse webhook connection method.
+
+Configure log collection for the Dataminr Pulse App using the below steps:
+
+1. Configure a Sumo Logic HTTP [Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source). The unique endpoint for receiving log and metric data for that source will be generated.
+2. Use Sumo Logic endpoint to push the data to Sumo Logic.
+3. Configure Webhook Delivery settings in Dataminr webhook platform:
+    1. Work with Dataminr Product Owner to get required credentials to call Integration Settings API and list API.
+    2. Call List API which gives the watchlists associated to the user accounts.
```

Comment:
would be good to provide links to Call List API and Integration settings API if possible so that users know what request payload to pass and which url to use.

Reply:
We do not have links to the Call List API and Integrations settings API. We will update the readme once the doc for the same gets published.